resources/public/api/conf/1.0/scenarios/amendPensionCharges.md

```diff
@@ -11,14 +11,6 @@
             <td><p>N/A - DEFAULT</p></td>
             <td><p>Simulates a success response.</p></td>
        </tr> 
-        <tr>
-            <td><p>FORMAT_NINO</p></td>
-            <td><p>Simulates a scenario where the format of the supplied NINO field is not valid.</p></td>
-        </tr>
-        <tr>
-            <td><p>FORMAT_TAX_YEAR</p></td>
-            <td><p>Simulates a scenario where the format of the supplied Tax year field is not valid.</p></td>
-        </tr>
         <tr>
             <td><p>MATCHING_RESOURCE_NOT_FOUND</p></td>
             <td><p>Simulates a scenario where the supplied income source could not be found.</p></td>
```

Comment:
A lil unrelated but just for consistency sake could you change the name for the gov test scenario to NOT_FOUND just to match up with our other apis as well as have each endpoint have matching not found scenarios